app/src/app/env.js

```diff
@@ -1,9 +1,13 @@
 import {env} from '../../package.json'
 import rc from './rc'
 
-if (rc.domain) {
+if (Object.prototype.hasOwnProperty.call(rc, 'domain')) {
```

Comment:
Why do you need to check own property here?

Reply:
when I just check `if(rc.domain)` the domain might be in the `rc` but might be evaluated to false anyway. e.g. domain is "false" or "0" or something like that.

Reply:
In javascript, `if (rc.domain)` is a value access.  If it is false or 0 or null or undefined you still have an empty value and if block won't be executed. I don't understand what you are doing here.

Reply:
this was changed to programm more definsive and for this part of the code to be more consistens (easier to read, easier to understand). the `if` block should be executed if the property is set, regerdless of the value. in this case (domain string) it doesn't help, but for true/false properties this is important. since we have two properties now already, the checking for the property should be consistent imho, to avoid mistakes in the future (e.g. someone adds another property and looks at the existing code).

i can change the whole thing to 

```
if (rc.domain) {
  env.host.domain = rc.domain
  env.chooseDomainDisabled = true
}

env.startInBackgroundWhenAutostarted = rc.startInBackgroundWhenAutostarted || env.startInBackgroundWhenAutostarted;
```

which is more consise but harder to extend and harder to understand at a glance. (compared to https://github.com/ubergrape/grape-electron/blob/ad991ae821e2f442b649a2465b889799ac55f068/app/src/app/env.js)